doc/RowBasedReplication.md

```diff
@@ -116,6 +120,24 @@ We have future plans to:
 * Maintain a history of the schema changes that happen on all shards, so events
   can be parsed correctly in all cases.
 
+## Unsupported Features
+
+This part describes the features that are not supported for RBR in Vitess:
+
+* *Fractional timestamps for MariaDB*: not supported. This affects the objects
+  of type `TIMESTAMP`, `TIME` and `DATETIME`. The way that feature is
+  implemented in MariaDB, the binary logs do not contain enough information to
+  be parsed, but instead MariaDB relies on the schema knowledge. This is very
+  fragile. MySQL 5.6+ added new data types, and these are supported.
+
+* *JSON type in MySQL 5.7+*: the representation of these in the binlogs is a
```

Comment:
We use JSON columns pretty widely, but we'd be fine with RBR launching first without it and having it supported in the next wave.